mantis-common/src/main/java/io/mantisrx/common/WorkerPorts.java

```diff
@@ -102,6 +103,16 @@ public List<Integer> getPorts() {
         return ports;
     }
 
+    @JsonIgnore
+    public List<Integer> getAllPorts() {
+        final List<Integer> allPorts = new ArrayList<>(ports);
```

Comment:
nit: It might be better to use ImmutableList here. I'll take care of it.

Reply:
+1